lib/ansible/modules/network/eos/eos_config.py

```diff
@@ -115,7 +115,7 @@
         without first checking if already configured.
       - Note this argument should be considered deprecated.  To achieve
         the equivalent, set the C(match=none) which is idempotent.  This argument
-        will be removed in a future release.
+        will be removed in Ansible 2.7.
```

Comment:
removed_in_version for force should be 2.6. We follow deprecated version + 4 for removed_in_version.